src/main/kotlin/fi/metatavu/timebank/api/persistence/repositories/VacationsRepository.kt

```diff
@@ -0,0 +1,84 @@
+package fi.metatavu.timebank.api.persistence.repositories
+
+import fi.metatavu.timebank.api.persistence.model.VacationRequest
+import io.quarkus.hibernate.reactive.panache.Panache
+import io.quarkus.hibernate.reactive.panache.PanacheRepositoryBase
+import io.quarkus.panache.common.Parameters
+import io.smallrye.mutiny.coroutines.awaitSuspending
+import java.time.LocalDate
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+
+/**
+ * Manages VacationRequest JPA entity
+ */
+@ApplicationScoped
+class VacationsRepository: PanacheRepositoryBase<VacationRequest, UUID> {
+
+    /**
+     * Lists VacationRequests based on given parameters
+     *
+     * @param personId persons id in Forecast
+     * @param before LocalDate to retrieve requests before given date
+     * @param after LocalDate to retrieve requests after given date
+     * @return List of VacationRequests
+     */
+    suspend fun getVacationRequest(personId: Int?, before: LocalDate?, after: LocalDate?): List<VacationRequest> {
```

Comment:
Consider implementing an abstract repository class to avoid repetitive logic (as this method and the list method in `TimeEntryRepository`)